docs/docs/dev/cli-reference/cli/node/delete/index.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: delete
+---
+
+# Command: `node approve`
+
+The `bacalhau node delete` command offers administrations the ability to remove a node from the cluster using its unique identifier.
+
+## Description:
+
+Using the `delete` sub-command, administrators can remove a node from the list of available compute nodes in the cluster. This feature is necessary for the management of the infrastructure.
```

Comment:
for consistency with above:  Using the `delete` sub-command under the `bacalhau node` umbrella